src/peft/config.py

```diff
@@ -102,6 +82,9 @@ def from_pretrained(cls, pretrained_model_name_or_path, subfolder=None, **kwargs
             kwargs (additional keyword arguments, *optional*):
                 Additional keyword arguments passed along to the child class initialization.
         """
+        # Avoid circular dependency .. TODO: fix this with a larger refactor
```

Comment:
Most of the time, circular imports means that the Python modules or not well organized. I haven't checked exactly what the issue is here but maybe it would be addressed by the comment you made earlier today in the meeting.

Reply:
Yes I think moving the tuner classes into separate folders should help